### PR TITLE
Mapfish print - second steps

### PR DESCRIPTION
**Pull request by kalbermattenm**
This PR is meant to be merged into the mapfish_print branch.
It configures a raw print service and there is still a lot to do...

@voisardf, could you review the Python files (the rest is print-Jasper stuff). Be aware that all the comments in `lib\content.py` will be removed with the next upcoming PR (I hope to do so on Friday...)

Thanks

### CODE REVIEW

File: crdppf/lib/content.py [hunk 1]
```diff
@@ -1,12 +1,61 @@
...
+    # Get the raw feature BBOX
+    bbox = get_feature_bbox(idemai)
 
+    # Get the feature center
+    feature_center = get_feature_center(idemai)
+
+    # Get the print BOX
+    print_box = get_print_format(bbox, request.registry.settings['pdf_config']['fitratio'])
+
+    log.warning('Calling feature: %s' % request.route_url('get_property')+'?ids='+idemai)
```
Comment (voisardf) [inline crdppf/lib/content.py]: from the crdppf point of view you can get only one property at a time - so id would suffice... or am I missing the point?

File: crdppf/lib/geometry_functions.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+from crdppf.models import DBSession
+from crdppf.models import Property, PaperFormats
+
+from sqlalchemy.sql import func
+
+def get_feature_center(idemai):
+    """ Extract a feature centroid regarding its idemai attribute
+    """
+
+    geom = DBSession.query(Property.geom).filter(Property.idemai==idemai). \
```
Comment (voisardf) [inline crdppf/lib/geometry_functions.py]: hmm, shouldn't we accept all property geom's in return and rather raise an error if there is more than one result (which shouldn't be the case)?

File: crdppf/lib/wmts_parsing.py [hunk 3]
```diff
@@ -0,0 +1,142 @@
+# -*- coding: UTF-8 -*-
+
+from xml.dom.minidom import parseString
+
```
Comment (voisardf) [inline crdppf/lib/wmts_parsing.py]: do you want to keep the white lines here and on line 11?

**Comment by voisardf** [comment]
Python stuff looks good to me, just some small questions/thoughts. :)

File: crdppf/lib/content.py (hunk 1 above)
Comment (kalbermattenm) [inline crdppf/lib/content.py]: I will change that (but this was defined in one of my previous PR... For which I did not ask any review... bad boy...)

File: crdppf/lib/geometry_functions.py (hunk 2 above)
Comment (kalbermattenm) [inline crdppf/lib/geometry_functions.py]: Hmmm... yes... why not...

But how do you except the requested route to answer to clients ? Should we send back an error 500 ?

File: crdppf/lib/wmts_parsing.py (hunk 3 above)
Comment (kalbermattenm) [inline crdppf/lib/wmts_parsing.py]: I like to make thing clear when importing stuff, like making a difference between SQLAlchemy stuff and other Python libs...

I will review the whole thing to be sure not to have miss a thing...

**Comment by kalbermattenm** [comment]
@voisardf, I addressed your review.

I will merge this in order to be able to continue...

**Pull request merged**